solutions/USACO jan21-uddered-bronze.mdx

```diff
@@ -0,0 +1,36 @@
+---
+id: usaco-1083
+source: USACO Bronze 2021 January
+title: Uddered but not Herd
+author: Ananth Kashyap
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_prob1_bronze_jan21.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(|word|)$
```

Comment:
maybe `**Time Complexity:** $\mathcal{O}(|\texttt{word}|)$` is better?
![image](https://user-images.githubusercontent.com/63619830/114813038-1b966c00-9d66-11eb-9ebf-39e409df8175.png)